Add issue template (#26)
qaisjp/mta-amx: .github/ISSUE_TEMPLATE/function_request.md

---
name: Function request

> about: You find a function or functions in SAMP but it still doesn't work here? Please, write here.

Reviewer: ```suggestion
about: Tell us about a function that does not work here, but works in SA-MP
```
